src/httpClient/httpClient.ts

```diff
@@ -256,3 +302,10 @@ export interface HttpClientOptions {
    */
   retryOptions?: RetryOptions;
 }
+
+/**
+ * Log options object.
+ */
+export interface LogOptions {
+  logData: LogData[];
```

Comment:
Please help out with this property name. It is similar to log level, but it isn't that.

Reply:
It's a tough one. Alternatively, we could avoid this by having two settings - `enableResponseLogs: bool` and `enableRequestLogs: bool`

Reply:
I was thinking about that.. but an array of enums (or ideally objects lol) scales better for future use-cases.

But you had the `enable` word there.. So what about:
```
enabledLogs: LogTypes[]

enum LogTypes {
  responses,
  requests
}

Reply:
Done. @mmaruniak please let me know if you have more comments or approve please :)

Reply:
I don't think this setting needs to scale to such an extent, but yeah doesn't hurt